x/auth/types/stdtx.go

```diff
@@ -191,6 +193,12 @@ type StdSignature struct {
 	Signature     []byte           `json:"signature"`
 }
 
+// GetPubKey returns the embedded crypto.PubKey type.
+func (sig StdSignature) GetPubKey() crypto.PubKey { return sig.PubKey }
```

Comment:
nit: can we add a `NewStdSignature` function above and use that elsewhere (eg: on `types/test_common`)?

Reply:
sure we can